.github/workflows/builder.yml

```diff
@@ -41,11 +41,11 @@ jobs:
     - name: Sign the images
       run: |
           echo "${DOCKER_PASSWD}" | docker login -u ${{ steps.get_repo_owner.outputs.repo_owner }} --password-stdin ghcr.io
-          cosign sign --force --key=cosign.key ghcr.io/${{ steps.get_repo_owner.outputs.repo_owner }}/golang-cross-builder:${{ env.GOLANG_CROSS_TAG }}
+          cosign sign --force ghcr.io/${{ steps.get_repo_owner.outputs.repo_owner }}/golang-cross-builder:${{ env.GOLANG_CROSS_TAG }}
```

Comment:
need to add the permissions as well here, similar you did for the other job

Reply:
Ah, sorry, my bad, I forgot to add this. Thanks again ❤️